win32ss/user/ntuser/display.c

```diff
@@ -151,43 +151,25 @@ InitDisplayDriver(
     return pGraphicsDevice;
 }
 
-NTSTATUS
-NTAPI
-InitVideo(VOID)
+static
+VOID
+GdiUpdateGraphicsDeviceList(VOID)
 {
-    ULONG iDevNum, iVGACompatible = -1, ulMaxObjectNumber = 0;
-    WCHAR awcDeviceName[20];
-    WCHAR awcBuffer[256];
-    NTSTATUS Status;
     PGRAPHICS_DEVICE pGraphicsDevice;
-    ULONG cbValue;
+    ULONG iDevNum;
     HKEY hkey;
+    NTSTATUS Status;
+    ULONG ulMaxObjectNumber = 0;
+    ULONG cbValue;
+    WCHAR awcDeviceName[20];
+    WCHAR awcBuffer[256];
 
-    TRACE("----------------------------- InitVideo() -------------------------------\n");
-
-    /* Check if VGA mode is requested, by finding the special volatile key created by VIDEOPRT */
-    Status = RegOpenKey(L"\\Registry\\Machine\\System\\CurrentControlSet\\Control\\GraphicsDrivers\\BaseVideo", &hkey);
-    if (NT_SUCCESS(Status))
-        ZwClose(hkey);
-    gbBaseVideo = NT_SUCCESS(Status);
-    if (gbBaseVideo)
-        ERR("VGA mode requested.\n");
-
-    /* Open the key for the adapters */
+    /* Initialize all devices */
     Status = RegOpenKey(KEY_VIDEO, &hkey);
     if (!NT_SUCCESS(Status))
     {
         ERR("Could not open HARDWARE\\DEVICEMAP\\VIDEO registry key:0x%lx\n", Status);
-        return Status;
-    }
-
-    /* Read the name of the VGA adapter */
-    cbValue = sizeof(awcDeviceName);
-    Status = RegQueryValue(hkey, L"VgaCompatible", REG_SZ, awcDeviceName, &cbValue);
```

Comment:
Functionality has been removed?

Reply:
This was only doing a log.

Reply:
OK. Normally this value is present in the registry for each video driver that is vga compatible btw. (may be used potentially as vga fallback ?)

Reply:
Log kept in new version, so this code is also kept.